src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -615,18 +619,39 @@ private void LaunchOnlineHelp(Uri uriToLaunch)
                     "https");
             }
 
+            // we use this test hook is to avoid actually calling out to another process
+            if (InternalTestHooks.BypassOnlineHelpRetrieval)
+            {
+                this.WriteObject(string.Format(CultureInfo.InvariantCulture, HelpDisplayStrings.OnlineHelpUri, uriToLaunch.OriginalString));
+                return;
+            }
+
             Exception exception = null;
+            bool wrapCaughtException = true;
             try
             {
                 this.WriteVerbose(string.Format(CultureInfo.InvariantCulture, HelpDisplayStrings.OnlineHelpUri, uriToLaunch.OriginalString));
                 System.Diagnostics.Process browserProcess = new System.Diagnostics.Process();
-
 #if UNIX
                 browserProcess.StartInfo.FileName = Platform.IsLinux ? "xdg-open" : /* OS X */ "open";
                 browserProcess.StartInfo.Arguments = uriToLaunch.OriginalString;
                 browserProcess.Start();
 #elif CORECLR
-                throw new PlatformNotSupportedException();
+                // On FullCLR, ProcessStartInfo.UseShellExecute is true by default. This means that the shell will be used when starting the process.
+                // On CoreCLR, this property doesn't exist. To work around this, we check if there is a default browser in the system.
```

Comment:
> On CoreCLR, this property doesn't exist.

This comment is not accurate. This property does exist in CoreCLR, but it always has the value `false`.